homeassistant/components/owntracks/messages.py

```diff
@@ -39,11 +39,15 @@ def _parse_topic(topic, subscribe_topic):
     Async friendly.
     """
     subscription = subscribe_topic.split("/")
-    try:
-        user_index = subscription.index("#")
-    except ValueError:
+
+    user_index = -1
```

Comment:
Please use the standard default value.
```suggestion
    user_index = None
```

Reply:
:+1: Done